package.json

```diff
@@ -24,10 +24,15 @@
   },
   "homepage": "https://github.com/AbhishekPunj0812/javascript-server#readme",
   "dependencies": {
+    "@types/mongoose": "^5.7.37",
     "body-parser": "^1.19.0",
     "commitizen": "^4.2.2",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
+    "mongodb": "^3.6.2",
+    "mongoose": "^5.10.13",
+    "secret-key": "^1.4.3",
```

Comment:
Why this package is needed, If the unused remove this

Reply:
ok sir